internal/internal_event_handlers.go

```diff
@@ -623,7 +623,10 @@ func (wc *workflowEnvironmentImpl) GetVersion(changeID string, minSupported, max
 		// Also upsert search attributes to enable ability to search by changeVersion.
 		version = maxSupported
 		wc.decisionsHelper.recordVersionMarker(changeID, version, wc.GetDataConverter())
-		wc.UpsertSearchAttributes(createSearchAttributesForChangeVersion(changeID, version, wc.changeVersions))
+		err := wc.UpsertSearchAttributes(createSearchAttributesForChangeVersion(changeID, version, wc.changeVersions))
+		if err != nil {
+			wc.logger.Warn("Failed to upsert search attributes for change version", zap.Error(err))
+		}
```

Comment:
This looks like a code change, maybe mention it in the description

Reply:
Added